evalml/model_understanding/_partial_dependence_utils.py

```diff
@@ -318,8 +318,8 @@ def _partial_dependence_calculation(
                     X_eval.ww[variable] = ww.init_series(
                         part_dep_column,
                         logical_type=X_eval.ww.logical_types[variable],
+                        origin=X_eval.ww.columns[variable].origin,
```

Comment:
Ideally, we'd be able to init with the original column's schema (as we do in fast mode above), as that would make sure that we don't lose any woodwork typing info as we replace this column, but that's not something we can currently do with init_series, so for now we'll just explicitly set individual woodwork parameters, but I opened up a Woodwork issue to make that possible: https://github.com/alteryx/woodwork/issues/1573

Reply:
Should we also create a corresponding EvalML issue to update this once the WW issue is resolved so we don't forget about it?

Reply:
Created: https://github.com/alteryx/evalml/issues/3847